stylus/utilities/border.styl

```diff
@@ -2,27 +2,27 @@
 @require '../tools/mixins'
 
 bdrs-0()
-    border-radius rem(0)
+    border-radius rem(0) !important // @stylint ignore
```

Comment:
Is there an explanation of why the `!important` is needed ?

Reply:
On a pas mal de classe utilitaires qui sont en important, justement parce que ce sont des classes utilitaires et qu'elles doivent prendre la main sur le css de l'app. Et certaines classes ne le sont pas, ce qui fait qu'elles ne servent à rien dans le cas d'un div qui a déjà le style correspondant définit, puisque le css de l'app est chargé en dernier.